pkgs/servers/code-server-bin/default.nix

```diff
@@ -0,0 +1,54 @@
+{ pkgs ? import <nixpkgs> { } }:
+
+let
+  stdenv = pkgs.stdenv;
+  lib = pkgs.lib;
+  system = stdenv.hostPlatform.system;
+
+  rsync = pkgs.rsync;
+
+in stdenv.mkDerivation rec {
+  pname = "code-server-bin";
+  version = "3.11.1";
+  commit = "1sb5ar3wh3a7c97vj63khdfv0p7inymim5xc8jfxb5i1b1whnjyk";
+
+  src = pkgs.fetchurl {
+    url =
+      "https://github.com/cdr/code-server/releases/download/v${version}/code-server-${version}-linux-amd64.tar.gz";
+    sha256 = "${commit}";
+  };
+
+  nativeBuildInputs = with pkgs; [
+    nodejs
+    yarn
+    python
+    pkg-config
+    zip
+    makeWrapper
+    git
+    rsync
+    jq
+    moreutils
+  ];
+
+  configurePhase = "true";
+  buildPhase = "true";
+  installPhase = ''
+    ${rsync}/bin/rsync -av ./ $out
+    rm $out/lib/node
+    ln -fs ${pkgs.nodejs}/bin/node $out/lib/node
+  '';
+
+  meta = with lib; {
+    description =
+      "Run VS Code on any machine anywhere and access it in the browser.";
```

Comment:
```suggestion
    description = "Run VS Code on any machine anywhere and access it in the browser.";
```

Reply:
Which formatter do you use ? I use nixfmt to format my *.nix files, i just wondering why nixfmt do an newline here :thinking: